### PR TITLE
feat(projects): enrich cards, fix dup header

### LINKED ISSUE #914: feat(projects): enrich cards & fix duplicate header
**Problem.** The Projects page duplicates its title + primary button — a top-bar "+ New Project" plus an in-body "Projects" heading and a second "+ Add Project" button (different label, same action). Project cards are sparse (repo + type + added date only). Project Detail shows "Tasks (N)" above six status buckets that are all `0` when the tasks are Done/Cancelled, so a busy project looks broken/empty.

**Fix.** Remove the duplicate in-body header/button; unify the label to "New Project". Add per-card counts (active tasks, open PRs, last activity). On Project Detail, show Done/total or a real summary instead of six zeros.

**Acceptance.** One create button; cards show useful counts; project detail never shows all-zero buckets for a non-empty project.

<sub>↻ Body corrected to match title — umbrella #903 subissue bodies were shifted by one slot.</sub>

### PR DESCRIPTION
**Pull request by Automaat**
## Motivation

The Projects page duplicated its title + primary button (top-bar "+ New Project" plus an in-body "Projects" heading and a second "+ Add Project"), cards were sparse, and Project Detail showed six all-zero status buckets for a project whose tasks were all done — looking broken (issue #914).

## Implementation information

- Removed the Projects page's in-body header + duplicate button. The page title and the single "New Project" create action live in the app top bar (the empty state keeps its onboarding CTA).
- **Enriched cards** with active-task and open-PR counts plus last-activity (`activeTaskCount`, `openPRCount` via `reviewStore.byRepo`, `lastActivity`). Seed `reviewStore.load()` at startup so counts are populated before the GitHub page is opened.
- **Project Detail**: an active/done/cancelled (+other) summary line, and when no task sits in an active board column it shows a summary instead of six empty columns. Active membership is derived from actual board placement, so a task with an unknown/legacy status can't reintroduce the all-zero buckets.

A pre-PR adversarial review caught the unknown-status edge case (the original `total − done − cancelled` math would have left six zeros) and the PR-count cold-start — both fixed here, with tests.

## Open questions / scope

- The empty state shows its own "Add your first project" CTA alongside the top-bar button — two create affordances, but that's the standard empty-state pattern (mutually exclusive with the populated list).

Closes #914

> Changelog: feat(projects): enrich cards, fix dup header

### CODE REVIEW

File: frontend/src/pages/ProjectList.svelte [hunk 1]
```diff
@@ -1,28 +1,38 @@
...
+  }
+
+  /** Most recent task activity for a project, as a relative label (empty if none). */
+  function lastActivity(projectId: string): string {
+    let latest = ''
+    for (const t of taskStore.list) {
+      if (t.projectId === projectId && t.updatedAt && t.updatedAt > latest) latest = t.updatedAt
+    }
+    return latest ? timeAgo(latest) : ''
+  }
```
[the remark this answers is not in the document]
Comment (Automaat) [reply frontend/src/pages/ProjectList.svelte:41]: Fixed in f7cafa0 — lastActivity now compares numeric timestamps via Date.parse() instead of lexicographic string comparison.

File: frontend/src/pages/ProjectList.svelte [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
     onadd: () => void
   }
 
   const { onselect, onadd }: Props = $props()
+
+  const TERMINAL = new Set(['done', 'cancelled'])
+
+  function activeTaskCount(projectId: string): number {
+    return taskStore.list.filter((t) => t.projectId === projectId && !TERMINAL.has(t.status)).length
+  }
```
[the remark this answers is not in the document]
Comment (Automaat) [reply frontend/src/pages/ProjectList.svelte]: Fixed in f7cafa0 — activeTaskCount now counts board-column membership (the status set derived from BOARD_COLUMNS' status + includes), so folded statuses count as active but unknown/legacy statuses are excluded, matching Project Detail's board/summary view. Added a test with a 'mystery' status.
